geoutils/georaster.py

```diff
@@ -1073,7 +1079,9 @@ def save(self, filename, driver='GTiff', dtype=None, blank_value=None, co_opts={
                       dtype=save_data.dtype,
                       crs=self.ds.crs,
                       transform=self.ds.transform,
-                      nodata=self.ds.nodata, **co_opts) as dst:
+                      nodata=self.ds.nodata,
+                      compress=compress,
+                      tiled=tiled, **co_opts) as dst:
```

Comment:
Would it not be safer to check that compress and tiled are not set within co_opts?
Apparently, setting twice the same argument does not raise any error and simply the last value is used, e.g.:
`img.save('test.tif', co_opts={"compress":"LZW", "compress":"DEFLATE"})`
saves with compression DEFLATE.
Or this could be the normal behavior, if we assume that people would simply try to overwrite `compress` using `co_opts`...

Reply:
That sounds like perfectly normal behavior to me! What we want is "if a user sets something, any default gets overwritten", which is currently the case, right?

Reply:
Yes, that's mostly fine. It's just weird not to have a warning for duplicated options (it should be fixed in rasterio).
The only case I can see is if the user set both compress and co_opts with conflicting values. Then co_opt will override compress. But that should not happen if the user uses save correctly.
Let's accept it as it is for now!

Reply:
Then submit a PR to python ;) It's in python that the duplication is allowed!:
```python
In [1]: {"compress": "lzw", "compress": "deflate"}
Out[1]: {'compress': 'deflate'}

```

Reply:
Haha, I wasn't aware of that! But the issue is that the same argument can be passed multiple times to rio.open here. Anyway, it is ok as it is.